### ISSUE TITLE
Blazor still not debuggable

### ISSUE THREAD

**Issue by jtrinanes** [dotnet/aspnetcore#23795]
Hi,

I'm trying out Blazor for my new project and I still can't debug a Blazor webassembly app.

I reinstalled Visual Studio 2019 in my machine and created a sample Blazor app. Inserted a breakpoint in the Counter.razor component and hit F5 and I'm getting this:

![image](https://user-images.githubusercontent.com/44257372/87013062-ae651180-c1fc-11ea-9946-09781eca1c43.png)

I'm using the latest Blazor release and followed the Blazor debugging documentation and still wasn't working :(

**Comment by SteveSandersonMS** [comment]
Could you please specify which versions you have of the following?

- Windows
- Edge/Chrome
- Visual Studio (eg 16.6.1)

Thanks!

**Comment by jtrinanes** [comment]
I'm using Windows 10, Microsoft Edge browser version 83.0.478.61 and Visual Studio 2019 Community edition version 16.6.3

**Comment by SteveSandersonMS** [comment]
Thanks for the info! Those all should work ok. 

To help diagnose, could you try following these steps to debug C# in the browser? https://docs.microsoft.com/en-us/aspnet/core/blazor/debug?view=aspnetcore-3.1#debug-in-the-browser

If that works we know the issue is something to do with VS. Or if it doesn’t work then it might point to another cause. Please let us know if you get any particular error messages.

**Comment by jtrinanes** [comment]
Yup it works in the browser. Hopefully on your next releases you guys enable debugging within Visual Studio :)

**Comment by SteveSandersonMS** [comment]
Thanks for confirming that.

@captainsafia Do you have a recommendation for how best to diagnose what might be going wrong? I think maybe you have a set of steps people can follow to collect more log output for investigation - is that right?

**Comment by captainsafia** [comment]
Thanks for reporting this issue, @jtrinanes! It looks like you're running into a known issue we're working on with regard to timeouts when launching a debugging session.

Getting some additional logging would help us get more information to track down where the bottleneck is here. Logs for the debugging experience are stored in two places.

* A debugging session file stored in `%TEMP%/vscode-chrome-debug.txt`
  * Note: This file gets regenerated for each debugging session you start and only stores the logs from the most recent one.
* Our debugging proxy logs which you can collect by following [these instructions](https://github.com/dotnet/aspnetcore/issues/22012#issuecomment-637117329)

Relatedly: Can you reproduce this issue when debugging in VS Code?
